### ISSUE TITLE
Option to listen for DNS over TLS queries

### ISSUE THREAD

**Issue by triatic** [getdnsapi/stubby#201]
Would it be feasible to have Stubby listen for DNS over TLS queries as well as send them? That way it could act as a fully-fledged DNS over TLS proxy without any additional software.

**Comment by CameronNemo** [comment]
What is the use case for this? Does unbound not address some of the needs?

**Comment by triatic** [comment]
Yes, Unbound can do this, as can Knot-Resolver. Maybe Stubby could do it too.

**Comment by CameronNemo** [comment]
Well I do not know how much effort it would be to add such functionality. But I will say that what attracts me to Stubby is that it is a minimal tool designed for the most common use case. For more complicated cases there are alternatives, like the ones you mentioned. I would hope that scope creep does not jeopardize one of stubby's main differentiating factors compared to other resolvers.
